libs/dh/wholesale/feature-calculation-steps/src/lib/actors/dh-wholesale-actors.component.html

```diff
@@ -15,28 +15,22 @@
 limitations under the License.
 -->
 <ng-container
-  *rxLet="
-    energySuppliersForConsumption$;
-    let energySuppliers;
-    suspense: loading;
-    error: error;
-    errorTrigger: errorTrigger$
-  "
+  *rxLet="actors$; let actors; suspense: loading; error: error; errorTrigger: errorTrigger$"
 >
-  <ng-container *transloco="let t; read: 'wholesale.energySuppliers'">
+  <ng-container *transloco="let t; read: 'wholesale.actors'">
     <h5 class="watt-on-light--high-emphasis">
-      {{ t("amountOfResults", { length: (energySuppliers || []).length }) }}
+      {{ t("amountOfResults", { length: (actors || []).length }) }}
     </h5>
     <watt-table
       [resolveHeader]="t"
-      [activeRow]="getSelectedEnergySupplier(energySuppliers)"
-      [activeRowComparator]="isSelectedEnergySupplier"
-      [dataSource]="_dataSource(energySuppliers)"
+      [activeRow]="getSelectedActor(actors)"
+      [activeRowComparator]="isSelectedActor"
+      [dataSource]="actors | wattDataSource: _dataSource"
```

Comment:
Question: I am not against this pipe, but couldn't the problem be solved in the component doing something like below?

```ts
_dataSource = new WattTableDataSource<WholesaleActorDto>([]);
actors$ = this.store.actors$.pipe(exists(), tap(actors => this._dataSource.data = actors);
```

Reply:
Good point, I changed it and removed the pipe